MapboxDirections/MBIntersection.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+public class Intersection: NSObject, NSSecureCoding {
+    public var inIndex: Int?
```

Comment:
This is an awkward place for a preposition. How about `entranceIndex` and `exitIndex`?

Also, does `[Bool].Index` work as the type for this property?